ygen/genstate_test.go

```diff
@@ -1300,6 +1300,53 @@ func TestBuildDirectoryDefinitions(t *testing.T) {
 	}
 }
 
+func enumMapFromEntries(entries []*yang.Entry) map[string]*yang.Entry {
```

Comment:
Can you add docstring comments that cover what these functions do within the tests?